lib/internal/test_runner/runner.js

```diff
@@ -593,6 +593,7 @@ function run(options = kEmptyObject) {
     argv = [],
     cwd = process.cwd(),
     rerunFailuresFilePath,
+    hookSignal = false,
```

Comment:
```suggestion
    hookSignal = true,
```
please preserve the current behavior as the default behavior